src/prefect/settings.py

```diff
@@ -1586,10 +1586,6 @@ def default_cloud_ui_url(settings, value):
 Whether or not to enable experimental work queue status in-place of work queue health.
 """
 
-PREFECT_EXPERIMENTAL_ENABLE_PYDANTIC_V2_INTERNALS = Setting(bool, default=False)
```

Comment:
removed since the setting is not in use, and because its presence would break `test_experimental.py`